### DISCUSSION TITLE
How to get more category commercial building?

### DISCUSSION THREAD

**Discussion by ghost** [streetcomplete/StreetComplete#5275] [Feedback]
I get some issue for spesific verification building, i can't find some category in commercial building like bank, convenience store, restaurant, ect. How to get this? Thank you

**Comment by mnalis** [comment]
Under `commercial building` category, there is `Retail building` (with description  `includes buildings for restaurants, cafes, stores...`) which is likely what you want in that case. 

And if it is Commercial building, but not a [Retail](https://en.wikipedia.org/wiki/Retail) one (and not one of the other more specific matches either), be aware that you **can** select that generic category `Commercial building` as an final answer too - just select it and click `OK`.

Also, I would not worry too much about being too precise. That quests tags what is ***architectural style*** of  a building, and **not** what is that building being used for. E.g. if it is a old abandoned church that is being used as a pub nowdays, you should still answer `Religious building` / `Church building` on that quest.

To add missing shops/amenities etc. (i.e. what that building is actually being used for, and what most end-users will search for - like `pub`, `shop` etc), you need to use `Shops overlay` (click on the second icon from the left on top on the main screen with map, and choose `Shops`)

**Reply by ghost** [reply]
Thanks for the explanation @mnalis
